docs/index.adoc

```diff
@@ -428,9 +531,10 @@ If the importer re-enters `Configuration Mode` and reconfigures the parameters i
 The importer is responsible for calling `fmi2SetupExperiment` at least once and setting the parameters before `fmi2SetupExperiment` is called.
 
 If the FMU does not expose its XCP configuration variables as `structuralParameters` using FMI 3.0, or as `parameters` using FMI 2.0, the importer will not have control over whether the internal XCP service should be started.
-In this case, the default value of the `start` attribute of variable `org.fmi_standard.fmi_ls_xcp.EnableXcpOn[Tcp|Udp]Ip` determines if the XCP service is started.
+In this case, the default value of the `start` attribute of the variable with role `XCPService{TCP|UDP}Enable`, if it exists, determines if the XCP service is started.
+If no variable of this role exists, then the XCP service is always started.
 
-_[Note that it is not possible to manually change the `start` value for `org.fmi_standard.fmi_ls_xcp.EnableXcpOn[Tcp|Udp]Ip` in the `modelDescription.xml` file to change the behavior of the internal XCP service._
+_[Note that it is not possible to manually change the `start` value for any variable in the `modelDescription.xml` file to change the behavior of the internal XCP service._

```

Comment:
Only valid for variable EnableXcpOn[Tcp|Udp]Ip ... so "with role `XCPService{TCP|UDP}Enable`"

Reply:
The statement is true for all variables, this is not specific to the enable variables (consider ports, etc.).